docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc

```diff
@@ -0,0 +1,301 @@
+
+image::docs/images/aerohive-radius-pf.png[scaledwidth="100%",alt="RADIUS server configuration"]
+
+Click OK to save the server. You should see the RADIUS server in the list.
+
+Configure the Open SSID
++++++++++++++++++++++++
+On the left menu, click on SSIDs. Click the New button, and configure the SSID while following those information.
+
+image::docs/images/aerohive-ssid-public.png[scaledwidth="100%",alt="SSID open configuration"]
+
+Click OK to save the SSID.
+
+Configure the Secure SSID
++++++++++++++++++++++++++
+On the left menu, click on SSIDs. Click the New button, and configure the SSID while following those inforamtion.
+
+image::docs/images/aerohive-ssid-secure.png[scaledwidth="100%",alt="SSID secure configuration"]
+
+Click OK to save the SSID.
+
+Configure User Profile
+^^^^^^^^^^^^^^^^^^^^^^
+You will now need to create user profile matching your VLAN assignements, the "Default VLAN" have been created before to match the desired VLAN number.
+
+Create user profile as follow:
+
+image::docs/images/aerohive-user-profile-reg.png[scaledwidth="100%",alt="User profile registration"]
+
+image::docs/images/aerohive-user-profile-prod.png[scaledwidth="100%",alt="User profile production"]
+
+Configure SNMP
+^^^^^^^^^^^^^^
+You now need to configure the SNMP to add a valid read community string. Click on the Configure pane, and on Advanced Configuration, then Management Policies, then SNMP Assignements. Now click on New and follow this configuration.
+
+image::docs/images/aerohive-add-snmp.png[scaledwidth="100%",alt="SNMP configuration"]
+
+Configure RADIUS attribute
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To be able to use the "Roles by switch role" you will need to setup a specific RADIUS attribute. To do so in the Configuration pane got to Networks, click on the network you will use, that will bring you to the "Guided configuration" now choose your SSID and click 'Add/Remove' under User Profile configure now this section as follow:
+
+image::docs/images/aerohive-radius-attribute.png[scaledwidth="100%",alt="RADIUS attribute option"]
+
+
+Step 4: Configuration of Windows 7 client for DemoSecure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In `Control Panel\Network and Internet\Manage Wireless Networks`, click on Add.
+Click Manually create a network profile
+
+   Network name: DemoSecure
+   Security type WPA2-Entreprise
+   Encryption AES
+
+Next, change connection settings. On Security tab, click Settings, uncheck Validate server certificate.
+On the same tab click Configure and uncheck Automatically use my Windows logon name and password.
+Return on the Security tab and click on Advanced settings. On 802.1X settings, click on Specify authentication mode and select User authentication.
+On 802.11 settings uncheck Enable Pairwise Master Key (PMK) caching.
+Validate all the modifications and click on close.
```

Comment:
_Next, change connection settings. On 'Security' tab, click 'Settings', uncheck 'Validate server certificate'.
On the same tab click 'Configure' and uncheck 'Automatically use my Windows logon name and password'.
Return on the 'Security' tab and click on 'Advanced settings'. On 802.1X settings, click on 'Specify authentication mode' and select 'User authentication'.
On 802.11 settings uncheck 'Enable Pairwise Master Key (PMK) caching'.
Validate all the modifications and click on 'Close'._